### PR TITLE
feat(docs/natives/RENDER_SCRIPT_CAMS): Renamed and explained the 4th …

### PR DESCRIPTION
**Pull request by koketoo**

### CODE REVIEW

File: CAM/RenderScriptCams.md [hunk 1]
```diff
@@ -8,22 +8,16 @@ ns: CAM
...
-Setting ease to 1 will smooth the transition.  
-```
+Renders the cams the current script
+
+## Parameters
+* **render**: Whether to render the cams or not.
+* **ease **: If true it will make a smooth transition between the current camera and the new rendered camera.
+* **easeTime **: The in milliseconds for the transition to happen.
+* **easeCoordsAnim**: If false it will only do the smooth transition to the rotation and the coords will be set instantly.
+* **p4**: Doesn't seem to change anything.
```
Comment (PsychoShock) [inline CAM/RenderScriptCams.md:18]: Param section should go after the `added param`.

Comment (PsychoShock) [inline CAM/RenderScriptCams.md:18]: Try to use something more general and add what value is used by default in decompiled scripts.

**Review by PsychoShock — CHANGES_REQUESTED** [review]

File: CAM/RenderScriptCams.md (hunk 1 above)
Comment (koketoo) [reply CAM/RenderScriptCams.md:18]: It's just that, if ease is true this param will change the behaviour in the transition

File: CAM/RenderScriptCams.md [hunk 2]
```diff
@@ -8,22 +8,15 @@ ns: CAM
 void RENDER_SCRIPT_CAMS(BOOL render, BOOL ease, int easeTime, BOOL p3, BOOL p4);
```
Comment (colistro123) [inline CAM/RenderScriptCams.md:8]: Where's `easeCoordsAnim`, which is `p3`? This is why the checks are failing.

**Review by colistro123 — CHANGES_REQUESTED** [review]
Hey there, requested some changes.

File: CAM/RenderScriptCams.md (hunk 2 above)
Comment (koketoo) [reply CAM/RenderScriptCams.md:8]: Oh, now I see thanks

**Review by AvarianKnight — CHANGES_REQUESTED** [review]

File: CAM/RenderScriptCams.md [hunk 3]
```diff
@@ -5,25 +5,28 @@ ns: CAM
...
 
 ## Parameters
-* **render**: 
-* **ease**: 
-* **easeTime**: 
-* **p3**: 
-* **p4**: 
+* **render**: Whether to render the cams or not.
+* **ease **: If true it will make a smooth transition between the current camera and the new rendered camera.
+* **easeTime **: The in milliseconds for the transition to happen.
```
[the remark this answers is not in the document]
Comment (koketoo) [reply CAM/RenderScriptCams.md:20]: Commited the suggestion, still not passing the build idk why

[the remark this answers is not in the document]
Comment (koketoo) [reply CAM/RenderScriptCams.md]: Now yes, thanks!

**Pull request merged**